truenas/webui
# NAS-111289 / 21.08 / Properly unsubscribe from reporting.realtime

To test:
- Make sure stats keep coming in on dashboard page.
- Check to see no more reporting.realtime payloads arrive on websocket after leaving the dashboard page.
